### PR TITLE
Issue10

### PR DESCRIPTION
**Pull request by stephenpaul2727**
Test cases for the presentational components of the current code in develop branch written. Please review and commit changes before incorporating new features.

### CODE REVIEW

File: actionlist/__tests__/compActionIttemBodySpec.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
+import React from 'react';
```
Comment (PrashanthKumarM) [inline actionlist/__tests__/compActionIttemBodySpec.js]: Please correct the filename

**Review by PrashanthKumarM — COMMENTED** [review]
I think everything seems fine here except the filename. But I think we should wait for @burnumd for a better review.

**Comment by stephenpaul2727** [comment]
Changed the file name of the test file.

**Review by PrashanthKumarM — APPROVED** [review]

File: actionlist/__tests__/compActionItemBodySpec.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      lastApprovedDate: 'last approved date',
+      processInstanceStatus: {
+        label: 'process instance status',
+      },
+    };
+    const middlewares = [];
+    const initialState = {};
+    const mockStore = configureStore(middlewares);
+    const store = mockStore(initialState);
+    const wrapper = mount(
```
Comment (burnumd) [inline actionlist/__tests__/compActionItemBodySpec.js]: I know there were issues with Enzyme earlier. What did you have to do to get it to work?

**Review by burnumd — CHANGES_REQUESTED** [review]
Instead of naming the tests `comp*Spec.js` the `__tests__` directory should probably have a directory structure similar to the source directory for organization.

**Review by burnumd — APPROVED** [review]

**Pull request merged**